tools/apollo/create_or_update_organism.py

```diff
@@ -56,7 +56,7 @@ def str2bool(string):
         password = pwgen(12)
         returnData = wa.users.createUser(args.email, firstName, lastName, password, role='user')
         gx_user = AssertUser(wa.users.loadUsers(email=args.email))
-        if (not args.use_remote_user):
+        if not str2bool(os.environ['GALAXY_WEBAPOLLO_REMOTE_USER']):
```

Comment:
We could, but I feel like it would be less intuitive. An user would need to completly remove the variable fro a docker-compose file instead of just setting it to "False". If we check the value, we can make sure all cases are taken care of.